### ISSUE TITLE
Exception when file name has no extension

### ISSUE THREAD

**Issue by JetA2** [Nikhil1999/uri-to-file#10]
If a content URL resolves to a file name with no extension, an exception is thrown:

```
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): Failed to handle method call
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): java.lang.StringIndexOutOfBoundsException: String index out of range: -1
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at java.lang.String.substring(String.java:2064)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at in.lazymanstudios.uri_to_file.model.UriToFile.fromUri(UriToFile.java:36)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at in.lazymanstudios.uri_to_file.helper.UriToFileMethodCallHandler.onMethodCall(UriToFileMethodCallHandler.java:23)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:262)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$DartMessenger(DartMessenger.java:319)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at io.flutter.embedding.engine.dart.-$$Lambda$DartMessenger$TsixYUB5E6FpKhMtCSQVHKE89gQ.run(Unknown Source:12)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at android.os.Handler.handleCallback(Handler.java:938)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at android.os.Looper.loop(Looper.java:223)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at android.app.ActivityThread.main(ActivityThread.java:7656)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
E/MethodChannel#in.lazymanstudios.uritofile/helper(18560): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by jaroslawdabrowski** [comment]
I am facing the same issue, e.g. when selecting a file from google drive, it does not contains extension.

**Comment by Nikhil1999** [comment]
Hi @JetA2 @jaroslawdabrowski 
This issue is now fixed in v1.0.0 
Please use the latest version 

Closing this as fixed in v1.0.0
